app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>user page</h1>
\ No newline at end of file
+<h1> </h1>
+<a href="<%= users_path %>">
+  <button id="back-btn">⬅</button>
+</a>
+<ul>
+  <% @posts.each do |post| %>
+    <li>
+      <a href="<%= user_post_path(post.author, post) %>">
+        <div class="user-box">
+          <%= render partial: 'pages/user_photo', locals: { user: post.author } %>
+          <%= render partial: 'pages/user_details', locals: { user: post.author } %>
+        </div>
+      </a>
+    </li>
+  <% end %>
+</ul>
+<a href="<%= users_path %>" class="pagination">
+  <button class="btn" id="pagination">pagination</button>
+</a>
+</ul> 
```

Comment:
| Your Design | Microverse Design |
|-|-|
| ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/06b6603c-e055-4f12-8e2f-adc5cd2b64b5) | ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/4bdb19c4-1f29-4831-8c03-97f5d56d7e75) |


- [ ] It would be nice if you double-check the original design given by microverse and follow that design while making your views to make your UI professional and attractive 👍
- [ ] Kindly follow the below details 👍

1. Kindly make the image on the left side and the size should be similar as you can see in the given design 👍
2. Kindly organize the username and posts_counter on the left as you can see in the design 👀
3. Kindly focus on the comments_counter and likes_counter because they don't show any number 😨
4. Kindly focus on this thing because everytime a post is created then it shows the user profile twice the times 😨 Means if there are two posts in the post_index view then it would show the user details two times 🤔